frontend/src/core/codemirror/cells/extensions.ts

```diff
@@ -52,6 +52,7 @@ export function cellMovementBundle(
     toggleHideCode,
     aiCellCompletion,
   } = callbacks;
+  const variables = useVariables()
```

Comment:
you can use `store.get(variables)` and can likely push this into `goToDefinition`

Reply:
Will add, thanks for the tip